packages/web/src/common/store/queue/sagas.ts

```diff
@@ -117,17 +117,12 @@ function* handleQueueAutoplay({
   const length = yield* select(getLength)
   const shuffle = yield* select(getShuffle)
   const repeatMode = yield* select(getRepeat)
-  const isCloseToEndOfQueue = index + 1 >= length
-  const isOnlySongInQueue = index === 0 && length === 1
+  const isCloseToEndOfQueue = index + 9 >= length
   const isNotRepeating =
     repeatMode === RepeatMode.OFF ||
     (repeatMode === RepeatMode.SINGLE && (skip || ignoreSkip))
 
-  if (
-    !shuffle &&
-    isNotRepeating &&
-    (isCloseToEndOfQueue || isOnlySongInQueue)
-  ) {
+  if (!shuffle && isNotRepeating && isCloseToEndOfQueue) {
```

Comment:
What is the situation with shuffle actually? Id assume if we get to the end of the queue in that case we'd want same logic?

Reply:
shuffle mode basically jumps around to different parts of the queue so this logic wouldn't apply correctly. but yeah i wonder how we deal with that, prob worth qa'ing to see if we get a similar issue